action.yml

```diff
@@ -12,6 +12,9 @@ inputs:
   tag-name:
     description: 'The tag associated with the release that contains the asset to download'
     required: true
+  repository:
+    description: 'If not provided, defaults to the repo where the action is being used'
```

Comment:
We might need to update this to indicate that the user needs to supply the organization/repository and it defaults to using github.repository which is the organization/repository.